helix-sitemap.yaml

```diff
@@ -0,0 +1,52 @@
+version: 1
+
+sitemaps:
+  seo:
+    origin: https://www.adobe.com
+    lastmod: YYYY-MM-DD
+    languages:
+      au:
+        source: /hub/query-index-au.json
+        destination: /hub/sitemap/sitemap-au.xml
+        hreflang: en
+        alternate: au/{path}
```

Comment:
in other projects we use `/au/{path}` (leading slash), not sure, whether this will work without.